solutions/devicesimulation-nohub/armtemplate/template.json

```diff
@@ -162,14 +162,14 @@
     },
     "pcsReleaseVersion": {
       "type": "string",
-      "defaultValue": "DS-1.0.1",
+      "defaultValue": "DS-1.0.2",
       "metadata": {
         "description": "The release version is used for repoURL for reverse-proxy-dotnet and vmScriptUri"
       }
     },
     "pcsDockerTag": {
       "type": "string",
-      "defaultValue": "millenium",
+      "defaultValue": "DS-1.0.2",
```

Comment:
Do you want these default value to updated at release time or be hardcoded in master?

Reply:
Default values are hard coded here.